Source/Core/VideoCommon/UberShaderPixel.cpp

```diff
@@ -948,8 +941,8 @@ ShaderCode GenPixelShader(APIType api_type, const ShaderHostConfig& host_config,
   {
     // Instead of using discard, fetch the framebuffer's color value and use it as the output
     // for this fragment.
-    out.Write("  #define discard_fragment {{ {} = float4(initial_ocol0.xyz, 1.0); return; }}\n",
-              use_shader_blend ? "real_ocol0" : "ocol0");
+    out.Write(
+        "  #define discard_fragment {{ real_ocol0 = float4(initial_ocol0.xyz, 1.0); return; }}\n");
```

Comment:
Unless I'm reading it wrong, `real_ocol0` only exists if `framebuffer_fetch` is true?

Reply:
`initial_ocol0` is also only defined if `framebuffer_fetch` is true, and that was always there

I think in general, code around that bug is written assuming fbfetch will be available.

Reply:
Ah, missed that about `initial_ocol0`.  Yeah, I think it's fine as is.  Worse case scenario we get a bug report and deal with it then :D